src/components/Breadcrumb/_breadcrumb.scss

```diff
@@ -0,0 +1,35 @@
+@import '#scss/_vars';
+
+.#{$px}-breadcrumb {
+  @include text($body3);
+
+  font-weight: 400;
+
+  ol {
+    display: flex;
+    flex-direction: row;
+
+    li {
+      display: flex;
+      flex-direction: row;
+
+      a {
+        margin: 10px;
```

Comment:
Does the figma reference spacing tokens for this?

Reply:
I am not seeing any spacing tokens and do not see any property when I drill down to the component on Figma. This was me eyeballing it, should I reach out to get an agreed upon spacing?

Reply:
$margin-sm looks to be the closest.

Reply:
Confirmed with Ryan that $margin-sm at 1rem is correct